src/Bicep.LangServer.IntegrationTests/CompletionTests.cs

```diff
@@ -117,6 +117,7 @@ private async Task<Container<Diagnostic>> StartServerAndGetDiagnosticsAsync(stri
                 {
                     options.OnPublishDiagnostics(diags => diagnosticsListener.AddMessage(diags));
                 },
+                resourceTypeProvider: new AzResourceTypeProvider(),
```

Comment:
FYI this build failure is due to a change I made under #2222. You'll need to merge main, and then change `new AzResourceTypeProvider()` -> `AzResourceTypeProvider.CreateWithAzTypes()`

Reply:
Merged latest from main and resolved conflicts.

Reply:
Applied the change above, now getting `error CS0426: The type name 'CreateWithAzTypes' does not exist in the type 'AzResourceTypeProvider'

Reply:
Thanks Anthony!

Updated here- https://github.com/Azure/bicep/commit/7a6d8ca11331abff0120cd1a2ead8feebd76aaca